daml-foundations/daml-ghc/ghc-compiler/src/DA/Daml/GHC/Compiler/Options.hs

```diff
@@ -223,6 +232,7 @@ defaultOptions mbVersion =
         , optDebug = False
         , optGhcCustomOpts = []
         , optScenarioService = EnableScenarioService True
+        , optScenarioValidation = ScenarioValidationLight
```

Comment:
The default option should be full validation. Light validation is only for the IDE.

Reply:
I would argue that the IDE is the default.

Reply:
I.e., tests are the special case not the code that you ship to users.

Reply:
I'd argue that the safe option, which is full validation, is the default, particularly since `damlc test` should do that, and that performance tweaks we do for the UI is the exception.

Reply:
Fair enough, I’ve swapped the default.

Reply:
Cool. Thanks.